src/pages/Vaults/Vault/components/VaultCollateral/VaultCollateral.tsx

```diff
@@ -161,25 +161,6 @@ const VaultCollateral = ({
           />
         </>
       )}
-      {/* <Modal open={isModalOpen} )}>
-        {(vaultAction === 'deposit' || vaultAction === 'withdraw') && (
-          <CollateralForm
-            ranges={ranges}
-            score={collateralScore}
-            collateral={collateral}
-            collateralToken={collateralToken}
-            variant={vaultAction}
-          />
-        )}
-        {(vaultAction === 'issue' || vaultAction === 'redeem') && (
-          <IssueRedeemForm
-            variant={vaultAction}
-            collateralToken={collateralToken}
-            remainingCapacity={remainingCapacity}
-            lockedAmountBTC={lockedAmountBTC}
-          />
-        )}
-      </Modal> */}
```

Comment:
@tomjeatt @savudani8 
The other day we agreed to remove this code. At that time, in the process of working on issue and redeem stuff (`IssueRedeemForm`), I found that this code was a little misleading. This time again, I was debugging inside `CollateralForm`. :) So, I'd suggest dropping the redundant code.